### PR TITLE
Porfolio-menu-mobile

### PR DESCRIPTION
**Pull request by jorgegoco**
-Added the mobile-menu HTML and CSS content.
-Created the JavaScript folder for all the JavaScript snippets.
-Created the Event-Handlers for the "hamburger menu", "close button", and section locations. 

### Collaborators:

@tahseenzahra 
@Roseokpe

### CODE REVIEW

File: head_mobile_portfolio.css [hunk 1]
```diff
@@ -15,6 +15,35 @@ body {
   border: 0;
 }
 
+#menu-mobile {
+  display: none;
+  width: 100vw;
+  height: 100vh;
+}
+
```
Comment (zieeco) [inline head_mobile_portfolio.css:23]: **[REQUIRED CHANGES]**

> hey @ Team, great job with the mobile menu, it works perfectly fine, however, there are some issues that need to be addressed to match the Figma design. Please, implement the changes below:

- Your design is great however some part of it is missing according to the design from [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). Please, kindly observe the design, you will see a black line,  a background `background: #E5E5E5;` and `border-bottom-right-radius, border-bottom-left-radius`. The border is on the mobile menu while the background and the black line are actually in a parent container holding the mobile menu. As this part of the design is overlooked, however, it's part of the mobile menu design. Please kindly update your styles and HTML to add this missing part to your design.
- The mobile menu takes up 90% height of the parent container and the parent container has a height of 100% which takes up the whole device height. Right now, your design is not taking up the whole device height, i.e, a user can choose to ignore your mobile menu links and just scroll down your page which doesn't improve the user experience. Please, kindly fix this to improve the user experience by making sure they should click on the links so it takes them to the corresponding path in your portfolio.

**Refer to the video below**

https://user-images.githubusercontent.com/59397159/168801763-fbe4105a-d695-49db-9970-26205a9d6510.mp4

- Also, please compare your design from the video with that of the Figma design:
**Figma**

details

Comment (zieeco) [reply head_mobile_portfolio.css:23]: - [ ] **[REQUIRED CHANGES]**

- Please, kindly update your readme authors by adding your team to the authors header, those that contributed to your portfolio this lets future employers see that you're pair programming and teamwork by collaborating with other developers around the world.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team


Great work on the mobile menu 👏 
You've done well implementing it, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**